config/database.yml

```diff
@@ -3,20 +3,27 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
   encoding: unicode
+  host: <%= ENV['HAVEN_DB_HOST'] %>
```

Comment:
`postgresql` refers to a different machine, right?  In this change, you're setting the same value for both the AWS desploy script and the Raspberry Pi deploy script, both of which install a postgres instance on the local machine.  

What does the `host:` get set to if we leave this line out?  Should we instead consider saying something like `ENV['HAVEN_DB_HOST'] || 'localhost'`?  That way we can set it if needed but it doesn't need to be specified otherwise?